applications/mlflow/chart/mlflow/templates/deployment.yaml

```diff
@@ -0,0 +1,259 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "mlflow.fullname" . }}
+  {{- with (merge (.Values.mlflow.labels | default dict) (include "mlflow.labels" . | fromYaml)) }}
+  labels: {{- toYaml . | nindent 4 }}
+  {{- end }}
+  {{- with .Values.mlflow.annotations }}
+  annotations: {{- toYaml . | nindent 4 }}
+  {{- end }}
```

Comment:
Just a general comment here, we've found with Velero labels we sometimes want them on specific objects. Does the pattern allow for such a thing or it global or none?

Reply:
Right now this is a global or none. I haven't seen too many charts where you can independently label every single object that gets deployed somehow. A library chart being the exception :)

Reply:
That's fair, you might find you need it when you work on backup ;)